Add support API Key and ID

mrlt8 opened the pull request:
This adds support for the API ID and Key which are now required: mrlt8/docker-wyze-bridge#1163

Information on how to obtain the ID and Key: https://support.wyze.com/hc/en-us/articles/16129834216731-Creating-an-API-Key

nwithan8 commented:
Please address the linting errors. Looks like an erroneous / on the DonateLink element

nwithan8 commented:
Also, consider adding a Requires section pointing people to the GitHub for configuration details, including how to retrieve an API key

mrlt8 commented:
Thanks for the feedback, @nwithan8! I've addressed the linting error and added a Requires section. Please let me know if there's anything else I can do!
